outline/functions.md

```diff
@@ -12,171 +12,83 @@ http://clojurebridge.github.io/curriculum/outline/functions.html
 
 {% endcomment %}
 
-<section>
+<section ng-controller="NarrativeController">
```

Comment:
It looks like it changes how the details links work: https://github.com/clojurebridge-berlin/curriculum/blob/gh-pages/assets/script.js